XBVC/emitters/templates/c_interface.jinja2

```diff
@@ -332,6 +369,11 @@ int xbvc_send(void *msg, uint32_t msg_type)
     return ret;
 }
 
+int xbvc_send(void *msg, uint32_t msg_type, uint32_t *rnd_id_ret)
+{
+    return xbvc_send_response(msg, msg_type, 0, rnd_id_ret);
```

Comment:
is 0 a special number that isn't part of the generated ids? Could a dispatcher see 0 in this field and know it's not a response?